products/ubuntu2004/profiles/stig.profile

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-title: 'Canonical Ubuntu 20.04 LTS Security Technical Implementation Guide (STIG) V1R1'
+title: 'Canonical Ubuntu 20.04 LTS Security Technical Implementation Guide (STIG) V1R8'
```

Comment:
replace it with V1R9

Reply:
Replaced with v1r9 in https://github.com/ComplianceAsCode/content/pull/10738/commits/bb07816b54debc4629643cbb381b3c886baa5e31